portal/static/js/main.js

```diff
@@ -832,24 +832,33 @@ var tnthAjax = {
     },
     withdrawConsent: function(userId, orgId, params, callback) {
         callback = callback || function() {};
-        if (!userId) {
-            callback({"error": i18next.t("User id is required.")});
-            return false;
-        }
-        if (!orgId) {
-            callback({"error": i18next.t("Organization id is required.")});
+        if (!userId || !orgId) {
+            callback({"error": i18next.t("User id and organization id are required.")});
             return false;
         }
         params = params || {};
-        this.sendRequest("/api/user/" + userId + "/consent/withdraw",
+        var self = this, arrConsent = [];
+        this.sendRequest("/api/user/" + userId + "/consent", "GET", userId, params, function(data) {
+            if (data && data.consent_agreements && data.consent_agreements.length > 0) {
+                arrConsent = $.grep(data.consent_agreements, function(item) {
+                    var expired = item.expires ? tnthDates.getDateDiff(String(item.expires)) : 0; /*global tnthDates */
```

Comment:
I think `getDateDiff()` could be more robust if it worked on `Date()` objects (ie `new Date()`), but out of scope for this PR...